core/src/main/resources/struts-excluded-classes.xml

```diff
@@ -55,52 +55,52 @@
     <!-- constant name="struts.excludedPackageNamePatterns" value="^java\.lang\..*,^ognl.*,^(?!javax\.servlet\..+)(javax\..+)" / -->
     <!-- constant name="struts.devMode.excludedPackageNamePatterns" value="^java\.lang\..*,^ognl.*,^(?!javax\.servlet\..+)(javax\..+)" / -->
 
-    <!-- this is simpler version of the above used with string comparison -->
+    <!-- All classes within the following packages and their sub-packages are excluded -->
     <constant name="struts.excludedPackageNames"
               value="
-                ognl.,
-                java.io.,
-                java.net.,
-                java.nio.,
-                javax.,
-                freemarker.core.,
-                freemarker.template.,
-                freemarker.ext.jsp.,
-                freemarker.ext.rhino.,
-                sun.misc.,
-                sun.reflect.,
-                javassist.,
-                org.apache.velocity.,
-                org.objectweb.asm.,
-                org.springframework.context.,
-                com.opensymphony.xwork2.inject.,
-                com.opensymphony.xwork2.ognl.,
-                com.opensymphony.xwork2.security.,
-                com.opensymphony.xwork2.util.,
-                org.apache.tomcat.,
-                org.apache.catalina.core.,
-                org.wildfly.extension.undertow.deployment."/>
+                ognl,
```

Comment:
Whilst trailing periods are still compatible, I've updated this list to abide by the new convention